dotcom-rendering/src/model/json-schema.json

```diff
@@ -390,6 +393,9 @@
         },
         "filterKeyEvents": {
             "type": "boolean"
+        },
+        "mostRecentBlockId": {
+            "type": "string"
```

Comment:
This and some other properties crept in here as the schema update routine was not running before

Reply:
Should this be automatic?